client/X11/xf_client.c

```diff
@@ -1422,7 +1423,9 @@ static BOOL xf_post_connect(freerdp* instance)
 	update->SetKeyboardIndicators = xf_keyboard_set_indicators;
 	update->SetKeyboardImeStatus = xf_keyboard_set_ime_status;
 
-	if (settings->RedirectClipboard && !(xfc->clipboard = xf_clipboard_new(xfc)))
+	serverIsWindowsPlatform = settings->OsMajorType == OSMAJORTYPE_WINDOWS;
```

Comment:
nitpicking:
 `serverIsWindowsPlatform = (settings->OsMajorType == OSMAJORTYPE_WINDOWS);`

Reply:
Okay, actually not relevant since the assignment operators have the second lowest operator order. In any case, did that in the last push.